bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/auth/client/oauth2/AccessTokenResponse.java

```diff
@@ -27,7 +27,7 @@
  * 4.4.3 - Client Credentials Grant - Access Token Response
  *
  * @author Michael Bock - Initial contribution
- * @author Gary Tse - Adaptation for Eclipse SmartHome
+ * @author Gary Tse - Adaptation for openHAB
```

Comment:
Hm, I assume this cannot be changed as it will change the "history".
Gary Tse has done the adaption for Eclipse SmartHome, regardless if we migrate that code from Eclipse SmartHome to openHAB Core.